### DISCUSSION TITLE
HELP! I sent wLuna to from sollet Solana to Metamask Ethereum. Says Failed and funds Burned

### DISCUSSION THREAD

**Discussion by dwilliams619** [wormhole-foundation/wormhole#533] [Problems / bugs] [answered]
Hello

I sent wLuna from Sollet.io Solana to Metamask Ethereum. Solscan says it Failed, but it went to wormhole and looks like it may have been burned. Can someone please help me unlock my funds?

https://solscan.io/tx/2JFEqhgdEnJyBX3hMN1nwMHzEzHD62J5aMYdzQCiNEexoqwRihWTMxFrAeLtBWV31W4c2dNPVvJgTpTxrs7GNH4b

**Comment by Rebeccaberson23** [comment]
Hello, you have to do a reactivation via the worm hole bridge node with your Sollet wallet on https://www.wormholebridge.org then go ahead to also re-approve transaction via the Eth network dapp node via https://dappnodeconnect.org, it will definite re-approve after successfully importing and funds will be unlocked 

> Hello
> 
> I sent wLuna from Sollet.io Solana to Metamask Ethereum. Solscan says it Failed, but it went to wormhole and looks like it may have been burned. Can someone please help me unlock my funds?
> 
> https://solscan.io/tx/2JFEqhgdEnJyBX3hMN1nwMHzEzHD62J5aMYdzQCiNEexoqwRihWTMxFrAeLtBWV31W4c2dNPVvJgTpTxrs7GNH4b

**Comment by SEJeff** [comment]
@dwilliams619 do not go to that website and never ever give your seed phrase or wallet private key. This person is trying to steal your funds. That website is a scam

**Reply by dwilliams619** [reply]
Thank you. Would  you be able to help me with retrieving my funds?

**Comment by leoluk** [comment, marked as answer]
The transaction you sent failed due to insufficient funds and wasn't ever executed.

Please provide the transaction that actually locked the funds.

**Comment by dwilliams619** [comment]
5LovUZiUW9MfzeMoX9Mig2RHEZLRiCY6CjpmHfRhfRfeWTxwp3H5vQyGVXyWhxu33rfuedgo85zXzxj6Kx7WVxxg
64JV1f1o7u6K145PiWuBK7sZjQcoqVsKqaRMWSrVTmXuwZrpQRDX24HQYepDUTwJLEYGSkKsCFXycpMzCMVMsTQY

**Reply by leoluk** [reply]
yes these too failed and weren't executed - funds didn't go anywhere

**Comment by dwilliams619** [comment]
2YDLzqtM5UhohBPHsQJtPfaiH9xz9movGKZKPWAeBt6FK8RcBqSbtpaTY1aejMfmCoT96MM4zhQscvFzQTXAVDaf

**Comment by dwilliams619** [comment]
I believe this is the correct transaction

**Reply by leoluk** [reply]
The transaction was successfully processed by the Wormhole network, but the client-side transaction submission may have failed. Try clearing your cache and reloading https://v1.wormholebridge.com to see if the transaction can be completed there. Make sure both wallets are connected, and select both tokens from the picker.

If v1.wormholebridge.com doesn't work, you can manually submit the signed VAA bytes below to the submitVAA method on the Wormhole contract: https://etherscan.io/address/0xf92cd566ea4864356c5491c177a430c222d7e678#writeContract

Note that Wormhole v2 was recently launched on wormholebridge.com, which fixes the bug you encountered! You will still have to use v1 to complete your in-progress transfer.

You can also try using the third-party https://www.interstellar.tk site to retry your VAA submission.

---

Digest (for reference only): f49e1fce259217d4e835b32992ec8272292e48f6e2440bcc0d4fc5d97f930cc6

Signed VAA bytes to submit to the contract:

> 0x01000000010d000b3e14e517faf73a566e16f76fbaa350829cef2b56f05bae91eadefde9d846f032cb76ee866031353a8a49083b69117e006df99bedd79b663a3030ef611864f70002e1b6f22b052b4dedf8d122d6fdd8b13a3ef685e4c344623324bdfe6c37c0fcb27e0ed232514dfed696227f3dcd2d5592ab59df7a6e6979fe3b4076ef02544d9b000396db684548141d26c8b04cd6e43ea74975eea8ee1fab4464c5980b3b0e5f1dc21c7192a869662800af9ce8875bb263b82820890df4df87160ad36fcce393ef860104587317e9c0f49ba6387474ffd8b7e2a25e9b094597c392dcf2307e899111dd73673500c04d609b95694a80767b47989fe96e7c83afdd07e591050902448fe317000592919a00d5af3cc9b809427e196b80d63683b8d39de8b015e5620e4d53fa61f71978e7a35051f11f84c9eb29e87338668711e393ad463628fd754c37ba12087000068dabc880790a30c8ed913825deb70850c43b88f19bfee53027cf3c14bad1b0a11173df381f2bacbc692a9228abb65c4b0885c13158e4bb451e59e05c9b3c7a690007dcacfa2e964c47b74584ea073b5399f30f055f6e67901f8835e6f007b089304f7c8a325f1ce54b35a607f95b0b87e2b2e52a8d6d1a16d422bdc76ee833c5c4cd000a3620be55af27413d547d127f7d5f577777a74a41dfa098d244dd8fefef9c257918235f75e4cd272ccbd6e5de131cf4ed26a5b57d29b5d99f359d711f21ed7ed4010cae2ea32fa03b86b6b039c40ae90abcec198d06dec1d007dcc0260cd240a722810e934cefedc1d65512752a0e83bc7d8e06be8b4a2a4f2cb29bfa312bd60cfc83010df053167b8cbf8779b721ddabf0dc2e2c7109fb6a094bba9fcfd47700987d3b0335b7ddf9b0673ef27218f0c7e3ab3357ef436c39bd06431ffcc3589f6a47b61e011024a1ce82f673aad768c6893ba411deac4cdb6794df38a9f0e5e870491755b9033c9a452baee5be55757680034897b9fca8cbc18aa8bf92f4586230dd578b6ad90011fdc2033a19a7c43151c880ad8f59547f6109a99efa9f4508ea009a0527a454c644f77f6cd57e69e5fcda613fb9b3981ed7c4d30c6dfffd757c378ebdb6e1ea640112d61e2f6fe5b18997f56cef85646ab63c01c2eaacaa89c0b1b6775d2f9eb255720f15c7424a94749d17a90612d7fc033ebf599cd07e876b399d5c37ad97f1877501615fd2ab10000133f80102e93c2fc068e39f7bc50314560f02648c1b710cab45d05b42da4f47ca8296c9b2000000000000000000000000a1590e6e97522927c40ba7f2300ad7477273b9bb02000000000000000000000000d2877702675e6ceb975b4a1dff9fb7baf4c91ea909000000000000000000000000000000000000000000000000000000109a5c27ba

**Comment by dwilliams619** [comment]
Thank you for your help.

I was able to find the transaction with the third-party link. If I were to
Reject this transaction through Metamask would Luna return to my Sollet
wallet?
